FINGERPRINT_Toolbox_Overview.adoc

```diff
@@ -7,7 +7,11 @@
 == Overview
 This toolbox contains a description of the process, tools and materials needed to construct and use a selection of fingerprint PAIs. Not all PAIs are described as there is an almost unlimited number when considering variations in materials and methods that can be used.
 
-Fingerprint PAD testing is a combination of fingerprint data acquisition, mold making, cast preparation (the PAI) and PAI presentation to the sensor.
+Fingerprint PAD testing is a combination of fingerprint data acquisition, mold making, cast preparation (the PAI) and PAI presentation to the sensor. 
+
+A fingerprint mold is the template that is created from the fingerprint that will be attacked. There are many ways to create a mold, such as directly from the finger, or from a scanned fingerprint image that is printed on a PCB. A mold should be able to be reused repeatedly for the purposes of this testing.
```

Comment:
I'd like to remove the word "template" since it's commonly used to refer to the collection of bytes that represent extracted fingerprint features. Here's an idea. I don't like "container" either, but I'm struggling to think of a better word now.

```suggestion
A fingerprint mold is a physical container that is created from the fingerprint that will be attacked. There are many ways to create a mold, such as directly from the finger, or from a scanned fingerprint image that is printed on a PCB. A mold should be able to be reused repeatedly for the purposes of this testing.
```

Reply:
I have made an update with the term "physical master pattern" instead. I don't know if that is really any better.